apps/next-evans/styles/globals.css

```diff
@@ -1,3 +1,153 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&display=swap');
+
 @tailwind base;
 @tailwind components;
+
+:root {
```

Comment:
instead of whole-sale moving all this CSS over from the old evans, can you just pull over the part you need to make this feature work? 

i think it's possible there's some cruft in here that we won't need, and it will make it hard to know what we can delete once all this is copied over.  we may find that if we just pull over the bits of global css that we need, one at a time, that much of this file can be discarded, or maybe we can move some of this funtionality into tailwind utilities.

in general, as you do work on this migration, take the approach of copying over the absolute minimum required to get things working for every part of the task, try to avoid pulling over chunks that aren't immediately being used and are necessary. this will help us end up with a cleaner slate when we're done. 👍